### PR TITLE
Update blogs.yml

### PR DESCRIPTION
**Pull request by hshafaq7**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

**Review by isuruAb — COMMENTED** [review]
please resolve the travis failure

File: _data/blogs.yml [hunk 1]
```diff
@@ -222,3 +222,8 @@
   title: Flying with Fossasia
   img: arjun.jpg
   author_name: Arjun Pandey
+
+- url:https://mygeekyworld786.wordpress.com/2017/12/30/here-i-come-coding-in/
+  title:Here I come Coding In
+  img:
```
Comment (nikit19) [inline _data/blogs.yml:228]: Please upload the image and write the name of the image after `img : `

File: _data/blogs.yml [hunk 2]
```diff
@@ -222,3 +222,8 @@
   title: Flying with Fossasia
   img: arjun.jpg
   author_name: Arjun Pandey
+
+- url:https://mygeekyworld786.wordpress.com/2017/12/30/here-i-come-coding-in/
+  title:Here I come Coding In
```
Comment (nikit19) [inline _data/blogs.yml:227]: please use the exact format as given in the readme.md there should be a space after `:`

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please make these changes

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Fix Travis CI failures. :+1:

**Review by akshatnitd — COMMENTED** [review]
Please make the suggested changes by @nikit19 .

**Comment by harshit98** [comment]
Closing this pull request. Similar to #704

**Pull request closed without merging**